test/e2e/common/pod_resize.go

```diff
@@ -210,20 +215,39 @@ func verifyPodStatusResources(pod *v1.Pod, tcInfo []TestContainerInfo) {
 	for i, c := range pod.Status.ContainerStatuses {
 		csMap[c.Name] = &pod.Status.ContainerStatuses[i]
 	}
+
 	for _, ci := range tcInfo {
 		cs, found := csMap[ci.Name]
 		framework.ExpectEqual(found, true)
-		tc := makeTestContainer(ci)
-		framework.ExpectEqual(cs.Resources, tc.Resources)
+
+		// Fixed the testing error for container specifying memory only, added by chenw.
```

Comment:
please remove this comment line. the next line is sufficient to explain the min defaults